pkg/harvester/l10n/en-us.yaml

```diff
@@ -38,6 +38,22 @@ asyncButton:
     success: Restarted
     waiting: Restarting&hellip;
 
+dialog:
+  confirmExecution:
+    title: Are you sure?
+    andOthers: |-
+      {count, plural,
+      =0 {}
+      =1 { and <b>one other </b>}
+      other { and <b>{count} other </b>}
+      }
+    confirmName: "Enter <b>{nameToMatch}</b> below to confirm:"
+    protip: "Tip: Hold the {alternateLabel} key while clicking action to bypass this confirmation"
+    stop:
+      message: "Are you sure you want to continue stop the {type} {names}?"
+    pause:
+      message: "Are you sure you want to continue pause the {type} {names}?"
```

Comment:
Copy most of the wording from Rancher.
ref: https://github.com/rancher/dashboard/blob/36d8a069eee391f916ca97f6c2b0f04d1bbb8db6/shell/assets/translations/en-us.yaml#L4764-L4779